src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -67,27 +68,27 @@ def create_results_link(output_dir: Path, link_name: Path) -> None:
 def perform_post_processing(
     raw_output_dir: Path, final_output_dir: Path, artifact_path: Path
 ) -> None:
-    raw_results = load_data(raw_output_dir)
+    processed_results = load_data(raw_output_dir)
     # Generate all post-processing maps to apply to raw results
     artifact = Artifact(artifact_path)
-    maps = generate_maps(raw_results, artifact)
+    maps = generate_maps(processed_results, artifact)
 
     # Iterate through expected forms and generate them. Generate columns each of these forms need to have.
     for observer in FINAL_OBSERVERS:
         logger.info(f"Processing data for {observer}...")
         # Fixme: This code assumes a 1 to 1 relationship of raw to final observers
-        obs_data = raw_results[observer]
+        obs_data = processed_results[observer]
 
         obs_data = obs_data.drop(columns=COLUMNS_TO_NOT_MAP[observer])
         for column in obs_data.columns:
             if column in maps:
-                for target_column_name, column_map in maps.items():
-                    obs_data[target_column_name] = obs_data.map(column_map)
+                for target_column_name, column_map in maps[column].items():
+                    obs_data[target_column_name] = obs_data[column].map(column_map)
                 # This assumes the column we are mapping will be dropped
-                obs_data.drop(columns=column)
+                obs_data.drop(columns=column, inplace=True)
 
         logger.info(f"Writing final results for {observer}.")
-        obs_data.to_csv(final_output_dir / f"{observer}.csv.bz2")
+        obs_data.to_csv(final_output_dir / f"{observer}.csv.bz2", index=False)
```

Comment:
This is where the final results are written.  Should we process columns here?

Reply:
This will be updated in a future ticket to return the correct columns for each observer.